### PR TITLE
Add 2.8.5 release notes

### PR DESCRIPTION
**Pull request by rpeden**
This PR adds release notes and updates the announcement bar for Prefect 2.8.5.

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
 # Prefect Release Notes
 
+## Release 2.8.5
+
+### Enhancements
+- Add basic event instrumentation to blocks — https://github.com/PrefectHQ/prefect/pull/8686
```
Comment (madkinsz) [inline RELEASE-NOTES.md:6]: Experimental

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
...
+- Add an endpoint to retrieve data from the collection registry — https://github.com/PrefectHQ/prefect/pull/8685
+
+### Fixes
+- Add carriage return to `prompt_select_from_list` — https://github.com/PrefectHQ/prefect/pull/8705
```
Comment (madkinsz) [inline RELEASE-NOTES.md:10]:
```suggestion
- Fix `prefect cloud login` detection of "ENTER" on some machines — https://github.com/PrefectHQ/prefect/pull/8705
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,44 @@
...
+- Fix Kubernetes job watch timeout request error by rounding — https://github.com/PrefectHQ/prefect/pull/8733
+- Ignore fsspec `2023.3.0` during requirements installation — https://github.com/PrefectHQ/prefect/pull/8757
+- Use `PREFECT_API_URL` when initializing the events client — https://github.com/PrefectHQ/prefect/pull/8704
```
Comment (madkinsz) [inline RELEASE-NOTES.md:13]: Experimental

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,44 @@
...
+- Fix Deployment and Concurrency Limit pages tabs — https://github.com/PrefectHQ/prefect/pull/8716
+- Add tests for base exceptions and calls — https://github.com/PrefectHQ/prefect/pull/8734
```
Comment (madkinsz) [inline RELEASE-NOTES.md:15]: Maintenance
```suggestion
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,44 @@
...
+- Remove deployment flow run foreign key — https://github.com/PrefectHQ/prefect/pull/8684
```
Comment (madkinsz) [inline RELEASE-NOTES.md:16]: Arguably an enhancement not a fix; what's the user-facing effect? Improve deletion of deployments?

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,44 @@
...
+### Experimental
+- Refactor supervisor API to allow configuration — https://github.com/PrefectHQ/prefect/pull/8695
+- Consolidate `WorkItem` and `Call` classes — https://github.com/PrefectHQ/prefect/pull/8697
+- Refactor supervisors to interact directly with "Worker" threads — https://github.com/PrefectHQ/prefect/pull/8714
+- Add chaining to cancel contexts — https://github.com/PrefectHQ/prefect/pull/8719
+- Add portal abstract base for worker threads and supervisors — https://github.com/PrefectHQ/prefect/pull/8717
+- Fix bugs in supervisors implementation — https://github.com/PrefectHQ/prefect/pull/8718
+- Refactor concurrency module and add documentation — https://github.com/PrefectHQ/prefect/pull/8724
+- Update block event resource IDs to use block-document id instead of name. — https://github.com/PrefectHQ/prefect/pull/8730
+- UI updates for workers beta — https://github.com/PrefectHQ/prefect/pull/8732
```
Comment (madkinsz) [inline RELEASE-NOTES.md:27]: Beta cancelled
```suggestion
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,44 @@
...
 
+## Release 2.8.5
+
+### Enhancements
+- Add basic event instrumentation to blocks — https://github.com/PrefectHQ/prefect/pull/8686
+- Add an endpoint to retrieve data from the collection registry — https://github.com/PrefectHQ/prefect/pull/8685
+
+### Fixes
+- Add carriage return to `prompt_select_from_list` — https://github.com/PrefectHQ/prefect/pull/8705
+- Fix Kubernetes job watch timeout request error by rounding — https://github.com/PrefectHQ/prefect/pull/8733
```
Comment (madkinsz) [inline RELEASE-NOTES.md:11]:
```suggestion
- Fix Kubernetes job watch timeout request error by rounding floats — https://github.com/PrefectHQ/prefect/pull/8733
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,44 @@
...
+- Ignore fsspec `2023.3.0` during requirements installation — https://github.com/PrefectHQ/prefect/pull/8757
```
Comment (madkinsz) [inline RELEASE-NOTES.md:12]:
```suggestion
- Fix flow load errors by ignoring fsspec `2023.3.0` during requirements installation — https://github.com/PrefectHQ/prefect/pull/8757
```

File: RELEASE-NOTES.md (hunk 5 above)
Comment (madkinsz) [reply RELEASE-NOTES.md]: @rpeden are we going to include more details about the ux here?

Comment (rpeden) [reply RELEASE-NOTES.md]: yes, added in latest commit

**Review by madkinsz — APPROVED** [review]

**Pull request merged**